addon/components/polaris-breadcrumbs/breadcrumb.js

```diff
@@ -14,7 +14,7 @@ export default LinkComponent.extend({
 
   breadcrumb: null,
 
-  params: computed('breadcrumb.{content,route,models.@each.id}', function() {
+  params: computed('breadcrumb.{content,route,models.[]}', function() {
```

Comment:
@andrewpye fyi, forgot about it, this was the reason the test was skipped (it was failing)....fixed with this loosened dep
<img width="1201" alt="screen shot 2018-05-18 at 2 38 24 pm" src="https://user-images.githubusercontent.com/160955/40232881-59b6106a-5aa9-11e8-83e0-3a91b5acc264.png">

Reply:
Thanks for the heads-up @vladucu! Weird one, any idea what changed to cause this?

My thinking would be more that this was a test setup issue, but I'm happy with your fix too 👍